src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -29,19 +29,14 @@
 public class NewGameChooser extends JDialog {
   private static final long serialVersionUID = -3223711652118741132L;
 
-  // Use synchronization when accessing s_cachedGameModel, it is accessed by both
-  // the Swing AWT event thread and also background threads, which parses available
-  // maps in the background when a game is not playing
-  private static NewGameChooserModel cachedGameModel = null;
-  private static ClearGameChooserCacheMessenger cacheClearedMessenger;
 
   private JButton okButton;
   private JButton cancelButton;
   private JList<NewGameChooserEntry> gameList;
   private JPanel infoPanel;
   private JEditorPane notesPanel;
   private NewGameChooserModel gameListModel;
-  private NewGameChooserEntry choosen;
+  private NewGameChooserEntry chosen;
```

Comment:
opportunistic fix of this variable name, while we were modifying this class and lines that related to 'choosen'